Description: int(math.Inf(1)) is broken and causes a test failure on mips64.
Upstream: yes
Url: https://github.com/itchyny/gojq/issues/69
diff --git a/operator.go b/operator.go
index 6ebd77e..418c184 100644
--- a/operator.go
+++ b/operator.go
@@ -378,7 +378,7 @@ func funcOpMul(_, l, r interface{}) interface{} {
 		deepMergeObjects,
 		func(l, r interface{}) interface{} {
 			multiplyString := func(s string, cnt float64) interface{} {
-				if cnt <= 0.0 || int(cnt) < 0 || int(cnt) > maxHalfInt/(16*(len(s)+1)) {
+				if cnt <= 0.0 || math.IsNaN(cnt - cnt) || int(cnt) > maxHalfInt/(16*(len(s)+1)) {
 					return nil
 				}
 				if cnt < 1.0 {
